contracts/database/orm/factory.go

```diff
@@ -0,0 +1,12 @@
+package orm
+
+//go:generate mockery --name=Factory
+type Factory interface {
+	Count(count int) Factory
+	Raw() any
+	Create() error
+	CreateQuietly() error
+	Make() Factory
```

Comment:
S: Sorry, I thought about it again, we may need this method, we can use it like below:
```
var user models.User
facades().Orm().Factory().Make(&user)

var user []models.User
facades().Orm().Factory().Times(3).Make(&user)// TImes defaults to 1
```

```suggestion
	Make(any) error
```

Reply:
Done